### ISSUE TITLE
slotLabelInterval start on the hour

### ISSUE THREAD

**Issue by vuoriliikaluoma** [fullcalendar/fullcalendar#3941]
Currently if I have a slot label interval of 1 hour
and the calendar start on a time like 7:30
then the slot interval starts there as well.

Is there a way to have the slot labels start from 8:00 (in this example)?
If not; feature request?

The start time of the calendar is variable in our system,
the users set their own work hours.

**Comment by acerix** [comment]
If I understand, you are referring to the `agendaDay` view with `mintime: '07:30'`, and you would like to offset the slot labels so they don't start at the top.  So, it would look like the following example, but without every second label (ie. rows with ':30' would have no label).

http://jsfiddle.net/u9k97h88/

If that is that case, I think it would have to be a new feature, but you could maybe use a hack to hide some of the labels instead.

**Comment by vuoriliikaluoma** [comment]
Correct, I was referring to the agenda-views with a `mintime`.
The problem I have is that the `mintime` could be at any 10 minutes of an hour.
So we'd like to be able to have the slot label times only show on 0 minutes of the hour no matter the `mintime`.

_If this isn't possible, I do have some workarounds in mind such as lowering `mintime` to 0 minutes._

**Comment by acerix** [comment]
Yes, for the labels to be 7pm, 8pm, 9pm..., the best option is to start `mintime` on the hour.

If you want have those labels when `mintime` is not on the hour (eg. 7:10), would it work to have a new option like `slotLabelOffset` which would be a Duration before the labels start?

**Comment by arshaw** [comment]
`slotLabelOffset` seems to be a reasonable addition to the API

**Comment by huntzinger92** [comment]
Looks like this feature was never added, according to current docs. Just adding a +1 for this being a useful feature!
